### ISSUE TITLE
Keep tags / header visible when scrolling

### ISSUE THREAD

**Issue by mrvdb** [astroidmail/astroid#494]
I use tags to classify mails for certain actions. What these need to be follows from me reading the message; often to the end.

It happens quite a lot that I'm classifying twice by accident which, in a toggle based set up, has caught me a couple of times (i.e. marking with tag 'something' is bound to a key 'X' which toggles it on and off) This has always happened on longer mails where the tags are out of the view

It would be very helpful if the tags would stay visible during the reading.

**Comment by gauteh** [comment]
Yeah, good point, I have missed this as well. I think we discussed something similar here https://github.com/astroidmail/astroid/issues/180#issuecomment-268969402. 

This could probably be solved with pure HTML / CSS.
